<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>691</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sayı 1:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="txt_sayi1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sayı 2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="txt_sayi2"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_toplama">
       <property name="text">
        <string>Toplama</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_cikarma">
       <property name="text">
        <string>Çıkarma</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn_carpma">
       <property name="text">
        <string>Çarpma</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btn_bolme">
       <property name="text">
        <string>Bölme</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="btn_tumIslemler">
       <property name="text">
        <string>Tüm İşlemler</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="5">
      <widget class="QLabel" name="toplamaSonuc">
       <property name="text">
        <string>Sonuç</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="5">
      <widget class="QLabel" name="cikarmaSonuc">
       <property name="text">
        <string>Sonuç</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="5">
      <widget class="QLabel" name="carpmaSonuc">
       <property name="text">
        <string>Sonuç</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="5">
      <widget class="QLabel" name="bolmeSonuc">
       <property name="text">
        <string>Sonuç</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
